Makefile

```diff
@@ -0,0 +1,41 @@
+export HOST_UID=$(shell id -u)
+export HOST_GID=$(shell id -g)
+
+DOCKER_COMPOSE_FILE_APP := ./docker/docker-compose.dev.yml
+DOCKER_COMPOSE_APP  := docker-compose -f $(DOCKER_COMPOSE_FILE_APP)
+DOCKER_COMPOSE_RUN_APP  := docker-compose -f $(DOCKER_COMPOSE_FILE_APP) run --no-deps
+
+up: prepare
+	$(DOCKER_COMPOSE_APP) up
+
+down:
+	$(DOCKER_COMPOSE_APP) down
+
+clean: clean-containers clean-modules
+
+clean-containers:
+	@docker kill $$(docker ps -q) || : &&\
```

Comment:
This might be a bit too aggressive since would remove all the modules.

Reply:
This is, indeed, too aggressive. Such a script is not a good citizen.

* `docker-compose down` removes the relevant containers and volumes, so there is no reason for `docker rm` and `docker volume prune`
* `docker-compose images -q` is a better way to target only the relevant Docker images instead of dropping all the Docker images from the host

Reply:
I will changes if Spectrum decide that is worth investing in this PR :-)
Thanks, good tips.

Reply:
To be honest, in the era of containers, such a script is an equivalent of `rm -rf /*`.